view/fullmodel/index.css

```diff
@@ -27,7 +28,7 @@
   }
 }
 
-.fullmodel_button {
+.fullmodel .is-close3d {
```

Comment:
Adding styles to another block is a violation of BEM rules.

Blocks should be independent. But here you must to know `.button` styles to be sure how your extra styles will work.